fix get-kube-local.sh 2 wrong parameters.

isaacdong opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:
1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-->

**bug fix**
1. run ./get-kube-local.sh, and get this error.
>  ./get-kube-local.sh: line 60: 2: unbound variable
2. remove duplicate filename in docker run command: `--kubeconfig=${KUBELET_KUBECONFIG}`

**Special notes for your reviewer**:
code detail:
line 88: create-kubelet-kubeconfig "${KUBELET_KUBECONFIG}" 
line 59: function create-kubelet-kubeconfig() {
line 60:  local destination="${1}"

**Release note**:
<!--  Write your release note:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. If no release note is required, just write "NONE".
-->
```release-note
      NONE
```

isaacdong commented:
/check-cla

isaacdong commented:
/assign @roberthbailey

roberthbailey commented:
Other implementations of the `create-kubelet-kubeconfig` function take an apiserver address as the first argument and the destination as the second argument. Having dest as $2 is consistent with those. 

It might make more sense to fix the call site to be
```bash
create-kubelet-kubeconfig "http://localhost:8080" $KUBELET_KUBECONFIG
```
instead of making the function signature diverge from the other functions of the same name. WDYT?

isaacdong commented:
There are three funtion create-kubelet-kubeconfig in code base:
https://github.com/kubernetes/kubernetes/search?q=create-kubelet-kubeconfig&unscoped_q=create-kubelet-kubeconfig

Keep them consistent.
Good idea, PR updated.

roberthbailey commented on cluster/get-kube-local.sh:
```diff
@@ -55,8 +55,9 @@ function run {
-# Args: destination file path
+# Args: the IP address of the API server (e.g. "http://localhost:8080"), destination file path
 function create-kubelet-kubeconfig() {
+  local api_addr="${1}"
```
this var isn't used (the api addr is hard coded to localhost:8080 on line 75). You should either use the var or comment that it is ignored.

roberthbailey reviewed (commented):
Can you please squash your changes?

isaacdong commented:
/cc @roberthbailey changes have been squashed.

isaacdong committed:
fix create-kubelet-kubeconfig.

1. keep function signature consistent.
implementations of the create-kubelet-kubeconfig function take an apiserver address as the first argument and the destination as the second argument.
2. remove duplicate file name(/kubelet.kubeconfig) in --kubeconfig=${KUBELET_KUBECONFIG}/kubelet.kubeconfig

roberthbailey commented:
/ok-to-test

roberthbailey commented:
/cc @krzyzacy (since I think this script is used in the test-infra)

roberthbailey commented:
/lgtm
/approve
